### ISSUE TITLE
Bundle DLBCL data

### ISSUE THREAD

**Issue by Kdreval** [morinlab/GAMBLR.data#16]
This repo should have metadata, ssm, sv, and cnv from at least a few DLBCL cell lines (file size permitting) that will be used for the future development of the testing suite for GAMBLR updates. This data should be bundled in both grch37 and hg38 projections

**Comment by Kdreval** [comment]
Based on the N of somatic mutations I have selected these 5 cell lines to minimize the size of the bundled Rda files:
```
   Tumor_Sample_Barcode  `n()`
   <chr>                 <int>
 1 DOHH-2                22089
 2 SU-DHL-10             26855
 3 OCI-Ly10              30051
 4 OCI-Ly3               31532
 5 SU-DHL-4              32824
 6 Toledo                41104
 7 SU-DHL-5              41439
 8 SU-DHL-9              45539
 9 SU-DHL-6              46669
10 Karpas422             58100
11 HBL-1                 77281
12 MD903                 80134
13 WSU-NHL              145531
14 Farage               153818
15 HT                   212749
```

So the winner is:
```
  Tumor_Sample_Barcode `n()`
  <chr>                <int>
1 DOHH-2               22089
2 SU-DHL-10            26855
3 OCI-Ly10             30051
4 OCI-Ly3              31532
5 SU-DHL-4             32824
```

**Comment by Kdreval** [comment]
Addressed in #17
